packages/frontend/component/src/ui/slider/slider.tsx

```diff
@@ -0,0 +1,101 @@
+import type { KeyboardEvent, MouseEvent } from 'react';
+import { useCallback, useEffect, useMemo, useRef, useState } from 'react';
+
+import * as styles from './index.css';
+
+export interface SliderProps {
+  value?: number;
+  onChange?: (value: number) => void;
+  min?: number;
+  max?: number;
+  step?: number;
+  nodes?: number[]; // The values where the nodes should be placed
+}
+
+export const Slider = ({
```

Comment:
please impl based on https://www.radix-ui.com/primitives/docs/components/slider for better keyboard + a11y support

Reply:
I have tried using Radix-UI, but the thumb does not move proportionally, causing the stepNode and thumb to be misaligned. This issue is currently blocking me from using Radix-UI.